source/customization/menus.rst

```diff
@@ -1,5 +1,367 @@
-=====
-Menus
-=====
+=======
+ Menus
+=======
 
-No documentation is available about this topic.
+.. seealso::
+
+   :doc:`Reference for menus </reference/menus>`
+      Reference for menu creation.
+
+Sublime Text provides several menus
+that can be modified,
+for example, by adding menu items.
+
+
+File Format
+===========
+
+=============  ===========================================
+**Format**     JSON (with comments)
+
+**Extension**  ``.sublime-menu``
+
+**Name**       One out of the list of available menus.
+               See :ref:`menu-types` for the complete name list
+               and what menu they represent.
+
+**Location**   Any
+
+**Content**    A list of "menu item" objects.
+=============  ===========================================
+
+
+Example
+*******
+
+The following is an excerpt
+from the default :file:`Main.sublime-menu` file.
+
+.. code-block:: json
+
+   [
+       {
+           "caption": "Edit",
+           "mnemonic": "E",
+           "id": "edit",
+           "children":
+           [
+               { "command": "undo", "mnemonic": "U" },
+               { "command": "redo_or_repeat", "mnemonic": "R" },
+               {
+                   "caption": "Undo Selection",
+                   "children":
+                   [
+                       { "command": "soft_undo" },
+                       { "command": "soft_redo" }
+                   ]
+               },
+               { "caption": "-", "id": "clipboard" },
+               { "command": "copy", "mnemonic": "C" },
+               { "command": "cut", "mnemonic": "t" },
+               { "command": "paste", "mnemonic": "P" },
+               { "command": "paste_and_indent", "mnemonic": "I" },
+               { "command": "paste_from_history", "caption": "Paste from History" }
+           ]
+       }
+   ]
+
+
+Images
+******
+
+.. figure:: images/context_menu_default.png
+   :class: shadowed
+   :figclass: float-left
+   :figwidth: 50%
+
+   The default context menu in the editing area.
+
+.. figure:: images/context_menu_modified.png
+   :class: shadowed
+   :figclass: float-left
+   :figwidth: 50%
+
+   A modified context menu in the editing area.
+
+
+.. _menu-types:
+
+Available Menus
+===============
+
+The file name
+of a ``.sublime-menu`` file
+specifies the menu to be modified
+in the application.
+
+The following menus are available:
+
+========================   ===============
+     File/Menu Name          Description
+========================   ===============
+**Main**                   Main menu

```

Comment:
I would probably add a full stop at the end of each description, as some of them are quite long and include conjugated verbs (i. e. a full sentence).